lib/helpers.js

```diff
@@ -4,6 +4,7 @@
 // consideration. We want to avoid scope creep in this library that would require
```

Comment:
How does this know which Sentry instance? Do we need specific environment variables? Please add any necessary instructions to the README.